payload.go

```diff
@@ -137,7 +144,15 @@ func ProcessPayload(e []github.Event, u User) ([]ActivityPayload, error) {
 			if comment, ok := rawPayload["comment"].(map[string]interface{}); ok {
 				payload.Body = comment["body"].(string)
 			}
-			activityPayloads = append(activityPayloads, payload)
+
+			// Save the paylod to the db events table
+			err := payload.Create()
+			if err != nil {
+				//Assuming fail due to the unique constraint
+			} else {
+				activityPayloads = append(activityPayloads, payload)
+			}
+
```

Comment:
Also, we can change the if block to just be

    if err == nil {
      activityPayloads = append(activityPayloads, payload)
    }

since we dont care about the `err != nil` block.  I'd still leave the comment tho so we remember why we dont care